src/file-events/cpp/win_fsnotifier.cpp

```diff
@@ -149,18 +149,16 @@ void Server::handleEvents(WatchPoint* watchPoint, DWORD errorCode, const vector<
             // Got a buffer overflow => current changes lost => send INVALIDATE on root
             logToJava(INFO, "Detected overflow for %s", utf16ToUtf8String(path).c_str());
             reportChange(env, FILE_EVENT_INVALIDATE, path);
```

Comment:
That explains the Windows soak test failure - since we don't receive any more events after the overflow.

Reply:
Good.